docker-image/README.md

```diff
@@ -1,112 +1,109 @@
 # Docker Image Gallery
 Azure Distributed Data Engineering Toolkit uses Docker containers to run Spark. 
 
-Please refer to the docs for details [how to select a docker-repo at cluster creation time](../docs/12-docker-image.md).
+Please refer to the docs for details on [how to select a docker-repo at cluster creation time](../docs/12-docker-image.md).
 
-### Supported Base Images
-We support several base images:
-- [Docker Hub] jiata/aztk:<image-version>-spark2.2.0-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark2.1.0-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark1.6.3-python3.5.4
-- [Docker Hub] jiata/aztk:<image-version>-spark2.1.0-python2.7.13
-- [Docker Hub] jiata/aztk:<image-version>-spark1.6.3-python2.7.13
+## Supported Images
+By default, this toolkit will use the Vanilla Spark image, __aztk-base__. This image contains the bare mininum to get Spark up and running in Standalone mode.
 
-NOTE: Replace **<image-version>** with the version of the image you wish to use. For example: **jiata/aztk:0.1.0-spark2.2.0-python3.5.4**
+On top of that, we also provide two flavors of Spark images, one geared towards the Python user (PySpark), and the other geared towards the R user (SparklyR or SparkR).
```

Comment:
Keep it to Pyspark for now?

@brnleehng  Also, do we have good support for SparkR really?